### DISCUSSION TITLE
Order Placement - Error Handling

### DISCUSSION THREAD

**Discussion by MichaelStaehli** [OpenWealth/OpenWealth#167] [Q&A] [answered]
**Requestor:**
Fabian Rutishauser - ZKB

**Submission date:**
16.03.2023

**Priority Community:**
N/A

**Effort T-Shirt Size:**
N/A

**Description:**
Due to technical restrictions, some checks will result in HTTP Error instead of a valid response (200) with error messages. Should the respective value of the enum be included in the HTTP error message?

**Comment by TLusti** [comment]
@MichaelWuestSGKB you handle it like @FabianRuti, right?

**Comment by MichaelWuestSGKB** [comment, marked as answer]
@TLusti Correct, in these cases we include the error message in the commonErrorResponses of the HTTP message.
